src/network/graphQuery.js

```diff
@@ -112,8 +112,6 @@ class GraphQuery {
 
   async execute() {
     const query = this.build();
-    console.debug(query);
```

Comment:
add this back in. I use this one a lot. and it wont show up unless you have `verbose` selected